content/docs/csidriver/release/powerstore.md

```diff
@@ -8,16 +8,18 @@ description: Release notes for PowerStore CSI driver
 ### New Features/Changes
 
 - Added support for Kubernetes 1.23.
+- Added support to configure FSGroupPolicy
 
 ### Fixed Issues
 
 There are no fixed issues in this release.
 
 ### Known Issues
 
-| Issue                                                        | Resolution or workaround, if known                           |
-| ------------------------------------------------------------ | ------------------------------------------------------------ |
-| Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation | Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100 <br>|
+| Issue                                                                                                                                      | Resolution or workaround, if known                                                                                                                                                                                                                                                                                                      |
+|--------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Delete namespace that has PVCs and pods created with the driver. The External health monitor sidecar crashes as a result of this operation | Deleting the namespace deletes the PVCs first and then removes the pods in the namespace. This brings a condition where pods exist without their PVCs and causes the external-health-monitor sidecar to crash. This is a known issue and has been reported at https://github.com/kubernetes-csi/external-health-monitor/issues/100 <br> |
+| FSGroupPolicy may not work as expected without root privileges for NFS only<br/>https://github.com/kubernetes/examples/issues/260          | To get the desired behavior set "allowRoot: "true" in the storage class parameter                                                                                                                                                                                                                                                       |
```

Comment:
Better to keep fsGroupPolicy in same case everywhere.

Reply:
I agree

Reply:
fixed